readme.md

```diff
@@ -134,6 +134,26 @@ Works the same as `add`, but mutates the original moment by subtracting time.
 
 For more information about `moment#subtract`, check out <http://momentjs.com/docs/#/manipulating/subtract/>.
 
+### timezone
+
+Convert the date to a certian timezone
+
+```html
+<span>{{ date | moment('timezone', 'America/Los_Angeles', 'LLLL ss')}}</span>
+```
+
+**To use this filter you will need to pass `moment-timezone` through to the plugin**
+
+```js
+// main.js
+import moment from 'moment-timezone'
+
```

Comment:
You could add `import VueMoment form 'vue-moment'`.
Might save a newby some confusion.

Reply:
Thanks, how does that look?

Reply:
Looks great. Merge it!